### PR TITLE
moving the large files into git-lfs and rebasing lee's branch on the fix

### PR DESCRIPTION
**Pull request by lbergelson**
@LeeTL1220 This branch has all of your commits rebased on top of a move to git lfs. 

 Feel free to work on top of it, but don't squash the two first 2 commits because bad things *might* happen.

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@lbergelson What should I do about the previous branch (and pull request)?  Close and delete?

**Comment by LeeTL1220** [comment]
Closes #884 and obviates #936

**Comment by LeeTL1220** [comment]
@davidbenjamin Louis had to move the changes of my issue #884 branches into a new branch.  This is that pull request.  I did the comments you recommended from the previous pull request.  Add new ones here.

**Comment by LeeTL1220** [comment]
@davidbenjamin If any of the tests fail, bounce this back to me.

File: scripts/unsupported/reheader_bam/reheader_bam.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    for i,r in enumerate(bamfile_in):
+        if r.next_reference_name in contigs:
+            try:
+                s_mate = bamfile_in.mate(r)
+                samfile_out.write(r)
+            except ValueError as ve:
+                if ve.message.find("fetch called") != -1 or ve.message.find("fetching by") != -1:
+                    raise ve
+
+        if i % 100 == 0:
```
Comment (davidbenjamin) [inline scripts/unsupported/reheader_bam/reheader_bam.py]: Every 100 reads seems too frequent.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
         final Pair<Double, Double> concordance = calculateConcordance(filteredVcf, truthVcf);
         final double sensitivity = concordance.getLeft();
         final double fdr = concordance.getRight();
-        Assert.assertTrue(sensitivity > requiredSensitivity);
+        Assert.assertTrue("Sensitivity " + sensitivity + " was lower than required sensitivity of " + requiredSensitivity, sensitivity > requiredSensitivity);
```
Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: Isn't the order `assertTrue(condition, message)`?

**Comment by davidbenjamin** [comment]
@LeeTL1220 congratulations; that was a ton of work.

File: scripts/unsupported/reheader_bam/reheader_bam.py (hunk 1 above)
Comment (LeeTL1220) [inline scripts/unsupported/reheader_bam/reheader_bam.py]: 1000 makes you worry that it froze, though....

No action unless you really want it.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java (hunk 2 above)
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: Nope.  I was just as surprised.  But I saw the failures working properly during my debugging.

**Pull request merged**